tests/test_special_ops.py

```diff
@@ -1097,3 +1097,29 @@ def test_accuracy_kron(shape, dtype):
         res_out = torch.kron(inp1, inp2)
 
     gems_assert_equal(res_out, ref_out)
+
+
+@pytest.mark.contiguous
+@pytest.mark.parametrize("shape", SPECIAL_SHAPES)
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES + ALL_INT_DTYPES)
+def test_accuracy_contiguous(shape, dtype):
+    if shape[0] <= 2:
+        return
+    if dtype in FLOAT_DTYPES:
+        inp = torch.randn(shape, dtype=dtype, device=flag_gems.device)
+    else:
+        inp = torch.randint(
+            low=-10000, high=10000, size=shape, dtype=dtype, device=flag_gems.device
+        )
+    inp = inp[::2]
+    np.testing.assert_equal(inp.is_contiguous(), False)
```

Comment:
`assert inp.is_contiguous() == False` is more simple.

Reply:
Done